pkg/cli/alpha/internal/update/prepare_test.go

```diff
@@ -70,7 +70,7 @@ var _ = Describe("Prepare for internal update", func() {
 		projectFile = filepath.Join(tmpDir, yaml.DefaultPath)
 
 		config.Register(config.Version{Number: 3}, func() config.Config {
-			return &v3.Cfg{Version: config.Version{Number: 3}, CliVersion: "1.0.0"}
+			return &v3.Cfg{Version: config.Version{Number: 3}, CliVersion: "v1.0.0"}
```

Comment:
CliVersion has not the v.
I think would be nice we have  a test here without v as well.
However, as far I remember the e2e tests have it so 👍 


/lgtm
/approved